azurerm/internal/services/cdn/cdn_endpoint_resource.go

```diff
@@ -201,10 +203,20 @@ func resourceArmCdnEndpoint() *schema.Resource {
 
 			"tags": tags.Schema(),
 		},
+
+		SchemaVersion: 1,
+		StateUpgraders: []schema.StateUpgrader{
+			{
+				Type:    resourceArmCdnEndpoint().CoreConfigSchema().ImpliedType(),
+				Upgrade: migration.CdnEndpointV0ToV1,
+				Version: 0,
```

Comment:
actually this wants to be `1` since it's what this is migrating to

```suggestion
				Version: 1,
```

Reply:
From the comment of `Version`, it says:

> Version is the version schema that this Upgrader will handle, converting it to Version+1.

Also the [front door custom https configuration](https://github.com/microsoft/terraform-provider-azurerm/blob/1f6f71654814266ebb9428eb2aa8be68665fb42b/azurerm/internal/services/frontdoor/frontdoor_custom_https_configuration_resource.go#L74) is also setting it as `0` and setting the `SchemaVersion` to `1`.

Reply:
As talked offline, this shall be kept.